runtime/build.rs

```diff
@@ -1,5 +1,3 @@
-use std::fs::copy;
-use std::path::PathBuf;
 
```

Comment:
CI doesn't like the blank line at the beginning of this file.

Reply:
Fixed